test/misc/misc_version_check_test.go

```diff
@@ -94,16 +96,13 @@ func TestMisc_empty_version_check(t *testing.T) {
 			t.Fail()
 		}
 	})
-	// needs to be the same client for the whole test
-	cfg := &weaviate.Config{
-		Host:    "localhost:8080",
-		Scheme:  "http",
-		Headers: map[string]string{},
-	}
-
-	client := weaviate.New(*cfg)
-
 	t.Run("Weaviate is not live, perform live check", func(t *testing.T) {
+		cfg := &weaviate.Config{
+			Host:    "localhost:8080",
```

Comment:
maybe use another port than 8080 - we have a constant "NoWeaviatePort" or something like that somewhere

Reply:
Ahh, this should actually be the port returned by `GetPortAndAuthPw` so thanks for the call-out